web/src/App.jsx

```
@@ -5253,7 +5264,23 @@ function App() {
+                    <img src={beaconLogo} alt="Beacon" className="empty-logo" />
+                    <h3 style={{ marginBottom: '8px' }}>Welcome to Beacon Inbox</h3>
+                    <p style={{ maxWidth: '400px', textAlign: 'center', color: 'var(--muted)', marginBottom: '24px' }}>
+                      Your messages will appear here once your phone syncs.
+                      Open PulseLink on your Android device to start the sync.
+                    </p>
+                    <button
+                      className="primary-btn"
+                      onClick={requestPhoneSync}
+                    >
+                      Request Sync from Phone
+                    </button>
```

Biome (2.3.13)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)